nym-vpn-cli/src/commands.rs

```diff
@@ -24,31 +24,32 @@ fn pretty_build_info_static() -> &'static str {
     PRETTY_BUILD_INFORMATION.get_or_init(|| bin_info_local_vergen!().pretty_print())
 }
 
-#[derive(Subcommand)]
-pub(crate) enum Commands {
-    /// Run the client
-    Run(RunArgs),
-    // Import credential
-    //ImportCredential(ImportCredentialArgs),
-}
-
 #[derive(Parser)]
 #[clap(author = "Nymtech", version, about, long_version = pretty_build_info_static())]
 pub(crate) struct CliArgs {
     /// Path pointing to an env file describing the network.
     #[arg(short, long, value_parser = check_path)]
     pub(crate) config_env_file: Option<PathBuf>,
 
+    /// Path to the data directory of the mixnet client.
+    #[arg(long)]
+    pub(crate) config_path: Option<PathBuf>,
```

Comment:
Shouldn't this be part of the `ImportCredentialArgs`, and non-optional, since we unwrap it later?

Reply:
it's also used in the run command, for lookup up keys